README.md

```diff
@@ -109,6 +109,25 @@ cp -r BUILD_apple_all_static/INSTALL/lib/CoreKiwix.xcframework ../apple/
 You can now launch the build from Xcode and use the iOS simulator or
 your macOS target. At this point the xcframework is not signed.
 
+
+## Deployment
```

Comment:
Use newlines between headers and sections ; GH renderer is permissive but not all renderer are and text-only is clearer with newlines

Reply:
Thanks, I've updated it.